src/Random/Symbols.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+declare(strict_types=1);
+/**
+ * @author Pavel Stepanets <pahhan.ne@gmail.com>
+ * @author Artem Dekhtyar <m@artemd.ru>
+ */
+
+namespace ElegantBro\Stringify\Random;
+
+use ElegantBro\Interfaces\Stringify;
+use ElegantBro\Stringify\Cached;
+use ElegantBro\Stringify\FromCallable;
+use Exception;
+use InvalidArgumentException;
+use function count;
+use function preg_split;
+use function random_int;
+
+final class Symbols implements Stringify
+{
+    /**
+     * @var Stringify
+     */
+    private $value;
+
+    public function __construct(Stringify $alphabet, int $length)
+    {
+        $this->value = new Cached(
+            new FromCallable(
+                static function () use ($alphabet, $length) {
+                    if ($length < 1) {
+                        throw new InvalidArgumentException('Length must be more than 0');
+                    }
+                    $alphabetLen = count(
+                        $codeAlphabet = preg_split(
```

Comment:
maybe the alphabet should be already split? because alphabet it not only one glyph I think
https://repl.it/@tigokr/alphabet-spliting

Reply:
good idea